pkg/cmd/cli/cmd/newapp.go

```diff
@@ -160,10 +203,10 @@ func RunNewApplication(f *clientcmd.Factory, out io.Writer, c *cobra.Command, ar
 		}
 	}
 
-	if len(cmdutil.GetFlagString(c, "output")) != 0 {
-		return f.Factory.PrintObject(c, result.List, out)
-	}
+	return nil
+}
 
+func produceOutput(f *clientcmd.Factory, out io.Writer, result *newcmd.AppResult) error {
```

Comment:
I think something like 'createObjects' would be a more appropriate name. produceOutput sounds like you're printing something.